src/main/java/org/opensearch/security/OpenSearchSecurityPlugin.java

```diff
@@ -183,6 +183,9 @@
     private static final Logger actionTrace = LogManager.getLogger("opendistro_security_action_trace");
     private static final DeprecationLogger deprecationLogger = DeprecationLogger.getLogger(OpenSearchSecurityPlugin.class);
 
+    public static final String LEGACY_OPENDISTRO_PREFIX = "_opendistro/_security";
```

Comment:
Reuse constant definition in other places, for example test ENDPOINTS.

Reply:
Not sure what do you mean, these are generic string I defined here to be used in other places. How can I reuse ENDPOINTS here as they are different for for different APIs?

Reply:
`"_opendistro/_security"` (aka `LEGACY_OPENDISTRO_PREFIX`) and `"plugins/_security"` (aka `PLUGINS_PREFIX`) are common prefixes used in all ENDPOINTS.

Reply:
Please also review `ENDPOINTS`, I see definition that can be simplified as in `AuditApiActionTest` `CONFIG_ENDPOINT` can be constructed from `ENDPOINT`.

Reply:
I reuse these prefixes in all the REST API tests